src/test/compile-fail-fulldeps/proc-macro/attr-invalid-exprs.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// aux-build:attr-stmt-expr.rs
+// ignore-stage1
+
+//! Attributes producing expressions in invalid locations
+
+#![feature(proc_macro, stmt_expr_attributes)]
+
+extern crate attr_stmt_expr;
+use attr_stmt_expr::duplicate;
+
+fn main() {
+    let _ = #[no_output] "Hello, world!";
+    //~^ ERROR asdf
+
+    let _ = #[duplicate] "Hello, world!";
```

Comment:
@petrochenkov 
https://travis-ci.org/rust-lang/rust/builds/360794617#L1905
> 25:13: 25:25: macro expansion ignores token `,` and any following

So it seems that the parser currently doesn't support directly expanding into >1 expressions.